presto-hive/src/test/java/com/facebook/presto/hive/TestMergeJoinPlanPrestoOnSpark.java

```diff
@@ -211,7 +216,8 @@ public void testDifferentSortByKey()
                     anyTree(
                             mergeJoin(INNER, ImmutableList.of(equiJoinClause("custkey_l", "custkey_r")),
                                     Optional.empty(),
-                                    anyTree(sort(tableScan("test_join_customer3", ImmutableMap.of("custkey_l", "custkey")))),
+                                    exchange(ExchangeNode.Scope.LOCAL, ExchangeNode.Type.GATHER, ImmutableList.of(sort("custkey_l", ASCENDING, FIRST)),
```

Comment:
Explicitly expect a gather exchange, which was a round robin exchange before this fix